### PR TITLE
Clarify use of legacy synchronous replication with BDR

### PR DESCRIPTION
**Pull request by mwanner**
It seems synchronous_standby_names is not compatible with CAMO at all, not just with the CAMO partner node. Mark it as deprecated more globally and visibly.

### CODE REVIEW

File: product_docs/docs/pgd/5/choosing_durability.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: "Choosing durability"
 EDB Postgres Distributed allows you to choose from several replication configurations based on your consistency, availability, and performance needs:
 
 * Asynchronous
-* Synchronous (using `synchronous_standby_names`)
+* Synchronous (using `synchronous_standby_names`, deprecated)
```
Comment (PJMODOS) [inline product_docs/docs/pgd/5/choosing_durability.mdx:8]: We should not deprecate this, we don't have anything that's comparable performance wise, if user only cares about durability from APP POV, and not the consistency properties, they should be free to use this no?

Comment (PJMODOS) [reply product_docs/docs/pgd/5/choosing_durability.mdx:8]: Especially given we have no proper group commit benchmarks :)

Comment (mwanner) [reply product_docs/docs/pgd/5/choosing_durability.mdx:8]: Yes, unless in combination with CAMO, Eager, Group Commit. So maybe "deprecated" is too strong, then.

The durability chapter does recommend Group Commit instead, though.

Comment (mwanner) [reply product_docs/docs/pgd/5/choosing_durability.mdx:8]: Dropped this change.

**Pull request merged**